### DISCUSSION TITLE
Shall we have a Discord community ?

### DISCUSSION THREAD

**Discussion by 0xApp** [DaveSkender/Stock.Indicators#1304] [Help and support]
I just stumbled upon this library and it seems awesome, I am current in process of creating my own trading application for which a lot of things would be needed.

What I feel is that as I make progress I can contribute to this library as well in same time.

A discord channel with enthusiast developer can help a long way.

Let me know what you feel about this idea.

**Comment by DaveSkender** [comment]
❤️ would love to have more contributors!

Regarding chat, if you want to maintain a discord channel, I’d be happy to join.  We also have an existing seldomly used Slack workspace too, in addition to the Discussions here.

**Comment by 0xApp** [comment]
Anything is great, I am ok anywhere
